### DISCUSSION TITLE
backoff in minutes or seconds?

### DISCUSSION THREAD

**Discussion by sheggi** [laravel/framework#36533] [Q&A]
Hello friends

Laravel v8.32.1
PHP 8.0

I'm using the ThrottlesExceptions middleware for handling an unstable API with Jobs. Now I wondered why a failed Job was not being retried after the defined seconds with backoff(30). Although the documentations says that backoff time is in seconds, this middleware applies backoff time in minutes.

```
// Illuminate/Queue/Middleware/ThrottlesExceptions
/**
     * Specify the number of seconds a job should be delayed when it is released (before it has reached its max exceptions).
     *
     * @param  int  $backoff
     * @return $this
     */
    public function backoff($backoff)
    {
        $this->retryAfterMinutes = $backoff;

        return $this;
    }
```

I'd expect the backoff time to be in seconds as in the documentation explained and also to be consistent with the use of backoff in other places. But I'm not sure, if it was intended to update the documentations.

**Comment by golonix** [comment]
In `ThrottlesExceptions::handle()`:
```
    // ...
    $this->limiter->hit($jobKey, $this->decayMinutes * 60);

    return $job->release($this->retryAfterMinutes * 60);
```
Backoff is set in minutes, then in `handle` method, it is converted to seconds.

Documentation states:
```
However, you may specify the number of seconds such a job should be delayed by calling the backoff method when attaching the middleware to the job:
```

Clearly, there is a mistake in the docs, comparing to what the code does. 
You can propose to change the behavior (to set backoff in seconds) and make a PR with the changes. I think having this unified is a valid reason. On the other hand, you can propose a PR that fixes the documentation.
